tests/trainer/logging_/test_train_loop_logging.py

```diff
@@ -749,3 +746,36 @@ def validation_epoch_end(self, *_) -> None:
     train_data = DataLoader(RandomDataset(32, 64), batch_size=2)
     val_data = DataLoader(RandomDataset(32, 64), batch_size=2)
     trainer.fit(model, train_dataloaders=train_data, val_dataloaders=val_data)
+
+
+def test_no_batch_size_extraction_with_specifying_explictly(tmpdir):
```

Comment:
Maybe I have missed here. do we have test case when user set batch_size in `LightningModule.log` and our code did not spam warning for ambiguous batch?

Reply:
yes. 2nd log inside `training_step`
```
 self.log("epoch_log_val", log_val, batch_size=batch_size, ...)
```